Makefile

```diff
@@ -1,4 +1,15 @@
 
+SIMULATOR_DESTINATION := OS=17.5,name=iPhone 15 Plus
+
+define XCODEBUILD_CMD
+	@set -o pipefail && \
+		xcodebuild $(1) \
+		-scheme GutenbergKit \
+		-sdk iphonesimulator \
+		-destination '${SIMULATOR_DESTINATION}' \
+		| xcbeautify
+endef
```

Comment:
I could have extracted the scheme and SDK, too, but I thought they're less likely to change than the destination.